docs/panel/advanced/docker.mdx

```diff
@@ -0,0 +1,203 @@
+```sh
+docker compose logs panel | grep 'Generated app key:'
+```
+
+### Install
+Open the installer in your browser at `APP_URL/installer` to finish setting up the panel.
+
+:::note
+The first time the container starts after installing or updating, it will apply database migrations. This process may take several minutes, during which the panel will not be accessible.
+:::
+
+If you don't have an external database you want to use, select SQLite with the default location for your database. If you don't have an external redis server, select Filesystem for Cache Drvier, Filesystem or Database for Session Driver, and Database for Queue Driver. 
+
+For other configuration, such as UI options, CAPTCHA, email, backups and OAuth, head to the settings menu in the admin panel.
+
+### Stopping the Panel
+The panel will automatically restart if the container crashes or the host restarts.  If you need to non-destructively stop the panel for any reason, navigate back to the directory containing `compose.yml` and run:
+
+```sh
+docker compose down
+```
+
+### Uninstalling
+To uninstall the panel, navigate to the directory containing `compose.yml` and run:
+
+```sh
+docker compose down -v
+```
+
+:::danger
+    **This will permanently delete the panel and all associated data** including the SQLite database and your encryption key.
+:::
+
+## Advanced Options
+### Redis
+You can deploy a Redis container in the same `compose.yml` file for easy setup. This example starts a Redis container and auto-populates the Redis configuration, cache, session and queue options in the web installer to use it.
+
+Once you use the web installer, changes to the highlighted environment variables will not apply, and those lines can be deleted from the `compose.yml` file. See [Configuring with Environment Variables](#configuring-with-environment-variables).
+
+
+```yml
+services:
+  redis:
+    image: redis:latest
+    restart: always
+    networks:
+      - default
+
+  panel:
+    image: ghcr.io/pelican-dev/panel:latest
+    depends_on:
+      - redis
+    restart: always
+    networks:
+      - default
+    ports:
+      - "80:80"
+      - "443:443"
+    extra_hosts:
+      - "host.docker.internal:host-gateway"
+    volumes:
+      - pelican-data:/pelican-data
+      - pelican-logs:/var/www/html/storage/logs
+    environment:
+      XDG_DATA_HOME: /pelican-data
+      # highlight-start
+      CACHE_STORE: redis
+      SESSION_DRIVER: redis
+      QUEUE_CONNECTION: redis
+      REDIS_HOST: redis
+      REDIS_PORT: 6379
+      # highlight-end
+      APP_URL: "http://localhost"
+      ADMIN_EMAIL: "USEYOUROWNEMAILHERE@example.com"
+
+volumes:
+  pelican-data:
+  pelican-logs:
+
+networks:
+  default:
+    ipam:
+      config:
+        - subnet: 172.20.0.0/16
+```
+
+### MySQL
+Deploying the panel in Docker is still a work in progress, and the default Docker image does not currently support using MySQL databases. Instructions for using MySQL will be added here once it is supported.
+
```

Comment:
You just put in the mysql database info into the .env and it works.

```suggestion
```

Reply:
Not currently in the Docker image. https://github.com/pelican-dev/panel/issues/986

Reply:
I see that issue is fixed so I'll add docs for this soon.